packages/shell-chrome/src/backend.js

```diff
@@ -57,16 +57,23 @@ function handleMessages(e) {
           var splittedAttrs = e.data.payload.attributeSequence.split("*");
 
           for (index in splittedAttrs) {
-            attributes = attributes + '["' + splittedAttrs[index] + '"]';
+            attributes += '["' + splittedAttrs[index] + '"]';
           }
 
           var data = component.__x.getUnobservedData();
 
-          eval(`(data${attributes} = '${e.data.payload.attributeValue}')`);
-          component.__x.$el.setAttribute("x-data", `${JSON.stringify(data)}`);
+          // nested path descriptor, eg. array*0*property needs to update array[0].property
```

Comment:
Is there any reason we're using * as a depth delimiter instead of .?

Reply:
https://github.com/Te7a-Houdini/alpinejs-devtools/pull/23

Reply:
Ah, didn't see that PR!